modules/artifact/pom.xml

```diff
@@ -15,12 +15,20 @@
         <!-- open internal required stuff from org.mule.runtime.core to the test infrastructure -->
         <surefire.args>
             -XX:+IgnoreUnrecognizedVMOptions
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.config.builders=ALL-UNNAMED
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.context=ALL-UNNAMED
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.el=ALL-UNNAMED
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.el.datetime=ALL-UNNAMED
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.registry=ALL-UNNAMED
-            --add-opens=org.mule.runtime.core/org.mule.runtime.core.internal.serialization=ALL-UNNAMED
+            --add-exports=org.mule.runtime.errors/org.mule.runtime.config.internal.error=ALL-UNNAMED
+              --add-opens=org.mule.runtime.core/org.mule.runtime.core.api.processor=ALL-UNNAMED
```

Comment:
This strange indentation is intentional?

Reply:
yes, to have the alingnment on the module name rather than the directive